iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/helpers/annotations/DeviceProvisioningServiceTest.java

```diff
@@ -0,0 +1,14 @@
+package com.microsoft.azure.sdk.iot.common.helpers.annotations;
+
+import java.lang.annotation.*;
+
+/**
+ * Tests that use the Device Provisioning Service
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.METHOD})
+@Inherited
+public @interface DeviceProvisioningServiceTest
+{
+
```

Comment:
No special rules for DPS tests to be defined here, but the DeviceProvisioningServiceTestRule class will check to see if a method/class is tagged with this annotation and run/skip the test accordingly

Reply:
Same for all these other annotations (other than ContinuousIntegrationTest which has a timeout as an additional rule)